supports/performance-test/mover-test/util.py

```diff
@@ -0,0 +1,436 @@
+import requests
```

Comment:
The util.py under _mover-test_/ looks as same as the one under _compression-test_/. We can create a folder named _util_ and put this script into it. Keeping one util.py can make the code maintenance easy in the future. Then, to make other scripts find util.py, you can **_export PYTHONPATH=../integration-test:$PYTHONPATH_** in shell script (refer to line 5 in _supports/ec-performance-test/test_ssm_ec_performance.sh_)

Reply:
OK, I will add "export PYTHONPATH=../integration-test:$PYTHONPATH" in shell script.